cico_run_coverage.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+. cico_setup.sh
+
+COVERAGE="f8osoproxy-coverage"
```

Comment:
It's probably a separate discussion but it would be great to move the code from our shell scripts to make so we can use it via make command in dev env. But I guess we want to keep the original make file untouched for easier upstream upgrades...

@aslakknutsen any thoughts?

But anyway it's not a blocker for this PR. Something for further improvements.

Reply:
@alexeykazakov we can do all this when we do https://openshift.io/openshiftio/Openshift_io/plan/detail/482 this will kind of separate oso_proxy from traefik and then oso_proxy will be like any other project we have in f8-services.

Reply:
Ah right... didn’t see that Work Item.
That WI is actually about sidecar container (one pod/deployment with two containers), right?
Just want to clarify what you mean by sidecar deployment.

Reply:
yes, one pod/deployment with two containers.